pkg/data/data.go

```diff
@@ -63,8 +63,15 @@ type KeySummaryStats struct {
 }
 
 // ValueJson gets the json representation of a kubernetes object stored in etcd.
+//
+// Deprecated: Use ValueJSON instead.
 func (ks *KeySummary) ValueJson() string {
-	return rawJsonMarshal(ks.Value)
+	return rawJSONMarshal(ks.Value)
```

Comment:
Should this call `ValueJSON` instead? To minimize drift in the implementation if, for some reason, `rawJSONMarshal` changes its behavior?

```suggestion
	return ks.ValueJSON()
```